templates/persistent-volume-claim.yaml

```diff
@@ -11,14 +11,10 @@ metadata:
     "app.kubernetes.io/component": {{ (index .Values.global.persistence.storageTierName 0) | quote }}
 spec:
   storageClassName: {{ .Values.global.persistence.storageClass | default "" | quote }}
-  {{- if .Values.global.localMount.enabled }}
   accessModes:
-    {{- range .Values.global.persistence.accessMode }}
-  - {{ . | quote }}
-    {{- end }}
+  - ReadOnlyMany
```

Comment:
are you sure we want ReadOnlyMany? I thought that would only allow for reads, however I would have thought that kafka and redis will also need to write values into the persistent volume?

Reply:
I don't think that's going to work.

Reply:
It's going to be changed by PAL-668 so I left it out here